exonum-java-binding/time-oracle/src/main/java/com/exonum/binding/time/TimeSchemaProxy.java

```diff
@@ -16,74 +16,74 @@
 
 package com.exonum.binding.time;
 
-import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.base.Preconditions.checkArgument;
 
 import com.exonum.binding.common.crypto.PublicKey;
 import com.exonum.binding.common.serialization.Serializer;
 import com.exonum.binding.common.serialization.StandardSerializers;
 import com.exonum.binding.core.storage.database.View;
 import com.exonum.binding.core.storage.indices.EntryIndexProxy;
+import com.exonum.binding.core.storage.indices.MapIndex;
 import com.exonum.binding.core.storage.indices.ProofMapIndexProxy;
-import com.exonum.binding.core.util.LibraryLoader;
+import com.exonum.binding.messages.Runtime.ArtifactId;
+import com.exonum.binding.messages.Runtime.InstanceSpec;
 import java.time.ZonedDateTime;
 
 class TimeSchemaProxy implements TimeSchema {
 
-  static {
-    LibraryLoader.load();
-  }
+  private static final int RUST_RUNTIME_ID = 0;
```

Comment:
Shouldn't this constant be somewhere in core?

Reply:
If it gets re-used, then yes; but currently can't see a good home for it (a new enum?)

Reply:
Enum in core sounds good, we'll probably need it somewhere else

Reply:
We could use that enum (and value of `JAVA_RUNTIME_ID`) in TestKit, for valid artifact ids creation (in case we don't want to do that in JNI - https://github.com/exonum/exonum-java-binding/pull/1161#discussion_r334222288 )

Reply:
We could consider creating CoreArtifactId (with the runtime id) and ServiceArtifactId (with no runtime id, to be used in metadata) so that the native does not have to perform these conversions. Or, drop the latter and always use four-component from `<runtime_id>:<group_id>:...` (even in JAR manifest).

Reply:
Or, drop the requirements on the format and just require `<runtime_id>:<artifact_id>`

Reply:
Now references RuntimeId.RUST.id